xforce.json

```diff
@@ -1,16 +1,16 @@
 {
     "appid": "653a4735-1eed-4cb9-98c4-b40408e60b22",
-    "name": "IBM XForce",
-    "description": "This app implements various 'investigative' actions on the 'IBM XForce' device",
+    "name": "IBM X-Force Exchange",
+    "description": "This app implements various 'investigative' actions on the 'IBM X-Force Exchange' device",
     "publisher": "AvantGarde Partners",
-    "package_name": "phantom_xforce",
+    "package_name": "phantom_ibmx-forceexchange",
     "type": "investigative",
-    "license": "Copyright (c) AvantGarde Partners 2017",
+    "copyright": "Copyright (c) AvantGarde Partners 2017",
```

Comment:
I think license is a required property in Splunkbase for package validation. At least until BASE-12986 is completed.

Reply:
Great catch! I added both `license` and `copyright` fields to make this robust.